source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRPresentationViewController.m

```diff
@@ -0,0 +1,44 @@
+//
+//  ACRPresentationViewController.m
+//  ADCIOS
+//
+//  Created by jwoo on 5/31/17.
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "ACRPresentationViewController.h"
+#import "ACRViewController.h"
+
+@implementation ACRPresentationViewController
+
+- (id) init:(NSString*) str{
+    self = [super init];
+    
+    if(self){
+        _jsonStr = nil;
+        _jsonStr = str;
+    }
+    
+    return self;
+}
+
+- (void) viewDidLoad {
+
+    [super viewDidLoad];
+    
+    if(self.jsonStr){
+        ACRViewController* acVc = [[ACRViewController alloc] init: self.jsonStr];
```

Comment:
>acVc [](start = 27, length = 4)

You should call `[self addChildViewController:acVc]` here, and then after adding the subview call `[acVc didMoveToParentViewController:self]`. This will establish a parent-child relationship between the two view controllers, which ensures that various system notifications will be forwarded from the parent to the child.